main/src/cgeo/geocaching/ui/dialog/CoordinatesCalculateDialog.java

```diff
@@ -542,6 +542,39 @@ private void displayToast(final int message, final boolean shortToast) {
         }
     }
 
+    private void setButtonInputValuesFromGP()
+    {

```

Comment:
"{" should be on the end of the previous line

Reply:
Hmm, funny Codacy only reported one (which I did fix).

Just did a RegEx search and yup, there were several, so I've fixed them as well now ;)